src/Cli/dotnet/commands/dotnet-workload/update/WorkloadUpdateCommand.cs

```diff
@@ -80,6 +82,22 @@ public override int Execute()
                 Reporter.WriteLine(workloadSet.ToJson());
                 Reporter.WriteLine("==workloadRollbackDefinitionJsonOutputEnd==");
             }
+            else if (!string.IsNullOrWhiteSpace(_workloadSetMode))
+            {
+                if (_workloadSetMode.Equals("workloadset", StringComparison.OrdinalIgnoreCase))
+                {
+                    _workloadInstaller.AdjustInstallMode(_dotnetPath, _sdkFeatureBand, "true");
+                }
+                else if (_workloadSetMode.Equals("loosemanifest", StringComparison.OrdinalIgnoreCase) ||
+                         _workloadSetMode.Equals("auto", StringComparison.OrdinalIgnoreCase))
+                {
+                    _workloadInstaller.AdjustInstallMode(_dotnetPath, _sdkFeatureBand, "false");
+                }
+                else
+                {
+                    throw new GracefulException(string.Format(LocalizableStrings.WorkloadSetModeTakesWorkloadSetLooseManifestOrAuto, _workloadSetMode), isUserError: true);
```

Comment:
I think if we use an Enum to represent the workload set mode (and use that enum in the option, ie `CliOption<WorkloadSetInstallMode>`), then System.CommandLine will handle this error checking for us.

Is this correct @baronfel?